presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/SinglenodePostgresql.java

```diff
@@ -71,7 +72,8 @@ private DockerContainer createPostgreSql()
                 .withEnv("POSTGRES_DB", "test")
                 .withEnv("PGPORT", Integer.toString(POSTGRESQL_PORT))
                 .withStartupCheckStrategy(new IsRunningStartupCheckStrategy())
-                .waitingFor(forSelectedPorts(POSTGRESQL_PORT));
+                .withHealthCheckCommand(format("pg_isready -t 1 -p %d -U ${POSTGRES_USER}", POSTGRESQL_PORT))
+                .waitingFor(forHealthcheck());
```

Comment:
nit: you can even add `waitingForHealthCheck()`.

Also be consistent with casing. IMO it should be `healthCheck` everywhere.

Reply:
forHealthcheck() is defined by testcontainers